vonk/releasenotes/releasenotes.rst

```diff
@@ -17,6 +17,44 @@ Upgrading Vonk
 
 See :ref:`upgrade` for information on how to upgrade to a new version of Vonk.
 
+.. _vonk_releasenotes_310:
+
+Release 3.1.0
+-------------
+
+Please also note the changes in :ref:`3.0.0-beta1 <vonk_releasenotes_300-beta1>` and in :ref:`3.0.0 <vonk_releasenotes_300>` (especially the one regarding the SQL server database)
+
+Fix
+^^^
+#. Validation on multi-level profiled resources no longer fails with the message `"Cannot walk into unknown StructureDefinition with canonical"`
+#. Improved documentation on :ref:`upgrading Vonk<upgrade>`, the :ref:`Vonk pipeline<vonk_reference_api_pipeline_configuration>`, :ref:`CORS support<configure_cors>`, :ref:`plugins<vonk_plugins>` and :ref:`IIS deployment<iis>`
+#. Using multiple parameters in _sort led to an error for all repositories
+#. Vonk UI capability statement view now works for self-mapped endpoints like ``/R3`` or ``/R4``
+#. A saved resource reference (e.g. ``Patient.generalPractitioner``) on a self-mapped endpoint (e.g. ``/R4/...``) would have its relative path duplicated (``/R4/R4/...``)
+
+   .. attention::
+
+      If you have used :ref:`self-mapped endpoints<feature_multiversion>` (appsettings: ``InformationModel.Mapping.Map`` in the 'Path' mapping mode) and you have saved resources containing references, it is possible that your database now contains some resources with broken references. Please contact us if this is the case
```

Comment:
Wondering whether this is actually the case: only the relative part should be stored in the database. Could you test that with Vonk 3.0.0 in these 2 cases:
- a path other than '/' in the PipelineOptions
- mapped version endpoints

Reply:
We have concluded the problem is real unfortunately, so I'm leaving this warning in.